stl/inc/format

```diff
@@ -1187,6 +1187,24 @@ public:
         }
     }
 
+    _NODISCARD size_t _Calculate_reserve_size() const noexcept {
+        using _CharT        = typename _Context::char_type;
+        size_t _Result      = 0;
+        const auto _Visitor = [&_Result]<class _ArgTy>(const _ArgTy _Arg) noexcept {

```

Comment:
Could this potentially copy user-defined arguments? It looks like this is a lightweight handle but I wanted to double-check.

Reply:
No, for user defined types _Arg is a [handle](https://github.com/microsoft/STL/blob/fc1b784ec863cfa91428384abcbc21bce27496ea/stl/inc/format#L184-L203).

Reply:
To be complete: `_Arg` is one of the alternative types of the exposition-only `variant` member of `basic_format_arg` (http://eel.is/c++draft/format.arg), all of which are small and (in our STL) trivially copyable,